src/plugins/process/README.md

```diff
@@ -0,0 +1,53 @@
+- infos = Information about the process plugin is in keys below
+- infos/author = e1528532 <e1528532@libelektra.org>
+- infos/licence = BSD
+- infos/needs = dump
```

Comment:
needs tells that dump should be loaded as separate plugin, this is not what we want